vm/src/tx_execution/system_sc.rs

```diff
@@ -0,0 +1,55 @@
+mod system_sc_issue;
+mod system_sc_special_roles;
+
+use crate::{tx_mock::{TxContext, TxResult}, types::VMAddress};
+use hex_literal::hex;
+use system_sc_issue::*;
+use system_sc_special_roles::*;
+
+/// Address of the system smart contract that manages ESDT.
+/// Bech32: erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u
+pub const ESDT_SYSTEM_SC_ADDRESS_ARRAY: [u8; 32] =
+    hex!("000000000000000000010000000000000000000000000000000000000002ffff");
+
+pub fn is_system_sc_address(address: &VMAddress) -> bool {
+    address.as_array() == &ESDT_SYSTEM_SC_ADDRESS_ARRAY
+}
+
+pub fn execute_system_sc(tx_context: TxContext) -> (TxContext, TxResult) {
+    let func_name = tx_context.tx_input_box.func_name.clone();
+    match func_name.as_str() {
+        "issue" => issue(tx_context),
+        "issueSemiFungible" => issue_semi_fungible(tx_context),
+        "issueNonFungible" => issue_non_fungible(tx_context),
+        "registerMetaESDT" => todo!(),
```

Comment:
I would replace these todos with a more explicit panic message

Reply:
Added blanket `fn`s for now with `unimplemented!()`